src/components/global/hooks/useDebounce.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react';
+
+// from https://usehooks.com/useDebounce/
+
+const useDebounce = <T,>(value: T, delay: number): T => {
```

Comment:
<T,> ? weird. prettier checks out though

Reply:
yeah i just lazily copied it...we have it in dac-ui too 🤷‍♀️